qa/L0_infer/infer_test.py

```diff
@@ -232,6 +232,17 @@ def _infer_exact_helper(tester,
                                         output1_raw=output1_raw,
                                         swap=swap)
 
+        if prefix == "":
```

Comment:
You should still use tu.validate_for_python_model here. It can just always return true.